cartridges/int_bolt_embedded_sfra/cartridge/scripts/hooks/payment/processor/bolt_pay.js

```diff
@@ -124,12 +125,20 @@ function authorize(orderNumber, paymentInstrument, paymentProcessor) {
         log.error(authRequestObj.errorMsg);
     }
 
+    // only attach oauth token if it is available and the user has not logged out
+    var boltOAuthToken = oAuth.getOAuthToken();
+    var bearerToken = null;
+    if (!empty(boltOAuthToken)) {
```

Comment:
@justinBolt1987 Do you know if it's possible to read the _bolt_sfcc_session_logout_ cookie here?

Reply:
done